internal/registration/tags.go

```diff
@@ -0,0 +1,80 @@
+package registration
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/pelletier/go-toml"
```

Comment:
I see there is one more [library](https://github.com/BurntSushi/toml) used in openshift. Both seems to have similar about of references. What was the reason you decided to use this one?

Reply:
No need at all, but I wanted to use the same code as Yggdrasil, we're going to remove this when the yggdrasil code got merged.